examples/rfq_quickstart/main.rs

```diff
@@ -179,6 +179,26 @@ async fn main() {
 
     // Stream quotes from RFQ stream
     while let Some(update) = rx.recv().await {
+        // Drain any additional buffered messages to get the most recent one
```

Comment:
can we add a big warning here please? if the user wants to track removed pairs, then they can not do this because the removed pairs are relative to the previous update! 
if they assume that wtv it is in the message is the current state (like we are doing in the quickstart) then this is fine